simpleui-server/src/simpleui-server.ts

```diff
@@ -88,14 +88,40 @@ export class SimpleUIServer {
                     const resourceEndIndex = link.url.indexOf('&', resourceIndex);
                     link.url = link.url.substring(0, resourceIndex + 9) + cmdVars.urlResource + link.url.substring(resourceEndIndex);
                 }
-            })
+            });
+        }
+        if (cmdVars.readonly) {
+            Logger.log(LogLevel.INFO, "Running app in readonly mode");
+            props['readonly'] = true;
+            // prepend 'Read Only ' to title of app
+            if (props['instance']['name'].substring(0,9) !== 'Read Only') {
+                props['instance']['name'] = `Read Only ${props['instance']['name']}`;
+            }
+            // prepend '(RO) ' to beginning of tab title
+            for (let i = 0; i < props['tab'].length; i++) {
+                if (props['tab'][i]['name'].substring(0,4) !== '(RO)') {
+                    props['tab'][i]['name'] = `(RO) ${props['tab'][i]['name']}`;
+                }
+
+                if (props['tab'][i]['disabled_buttons'] !== 'true') {
```

Comment:
Probably don't need the test - just always make the assignment?

Reply:
good point, just making the assignment is best. Just fixed this